control/control.leanos.xml

```diff
@@ -89,7 +89,7 @@ textdomain="control"
         <!-- Set SELinux disabled mode by default -->
         <selinux>
           <mode>disabled</mode>
-          <configurable config:type="boolean">true</configurable>
+          <configurable config:type="boolean">false</configurable>
```

Comment:
I would add the bug or jira reference also here.

Reply:
Done.